src/components/controls/PlusMinus.js

```diff
@@ -25,70 +21,21 @@ const Wrapper = styled.div`
   align-items: center;
   justify-content: space-between;
   border-radius: 3px;
-  background-color: #F9F9FD;
-  border: 1px solid #DCDCE7;
+  background-color: ${NEUTRAL.N50};
+  border: 1px solid ${NEUTRAL.N100};
   height: 40px;
   width: 180px;
 `;
 
-const getBackgroundColor = ({ disabled, selected } :any) => {
-  if (disabled && selected) return GRAY;
-  return selected ? PURPLE : WHITE;
-};
-
-const getColor = ({ disabled, selected } :any) => {
-  if (selected) return WHITE;
-  return (disabled) ? GRAY : PURPLE;
-};
-
-const Button = styled.button`
-  width: 33%;
-  height: 100%;
-  font-size: 14px;
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  align-items: center;
-  background-color: #F9F9FD;
-  color: #535253;
-  border: none;
-  font-family: Inter;
-  font-style: normal;
-  font-weight: normal;
-  font-size: 16px;
-  line-height: 22px;
-  /* identical to box height */
-
-  text-align: center;
-  pointer-events: ${(props) => (props.disabled ? 'none' : 'auto')};
-
-  &:hover:enabled {
-    cursor: pointer;
-    background-color: #dcdce7;
-  }
-
-  &:first-child {
-    border-radius: ${(props) => (props.selected ? 0 : '3px 0 0 3px')};
-  }
-
-  &:last-child {
-    border-radius: ${(props) => (props.selected ? 0 : '0 3px 3px 0')};
-  }
-
-  &:focus {
-    outline: none;
-  }
-`;
-
 const PlusMinus = ({ value, onChange } :Props) => (
   <Wrapper>
-    <Button disabled={!value} onClick={() => onChange(value - 1)}>
+    <IconButton disabled={!value} aria-labelledby="numberOfInfants" onClick={() => onChange(value - 1)}>
```

Comment:
Isn't the icon button round where as the one it's replacing is rectangular?

Reply:
The wrapper is the same color and there is no border on the icon button. So it appears circular on hover, but it's not offensive to me.